gemfire-core/src/main/java/com/gemstone/gemfire/internal/cache/LocalRegion.java

```diff
@@ -235,6 +235,8 @@
 
   public volatile boolean isDestroyed = false;
 
+  public volatile GemFireException daeException;
```

Comment:
If this is specific to ConflictingException then I think making the specific name diskConflictingException would be better.